internal/protobuf/testhelpers.go

```diff
@@ -11,35 +11,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package assert provides assertion helpers
-package assert
+// Package protobuf provides helpers for working with golang/protobuf types.
+
+package protobuf
 
 import (
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
-	tassert "github.com/stretchr/testify/assert"
 	"google.golang.org/protobuf/testing/protocmp"
 )
 
-var (
-	Equal         = tassert.Equal
-	Equalf        = tassert.Equalf
-	ElementsMatch = tassert.ElementsMatch
-	Error         = tassert.Error
-	Errorf        = tassert.Errorf
-	NoError       = tassert.NoError
-	NoErrorf      = tassert.NoErrorf
-	True          = tassert.True
-	Truef         = tassert.Truef
-	False         = tassert.False
-	Falsef        = tassert.Falsef
-)
-
-// EqualProto will test that want == got for protobufs, call t.Error if it does not,
+// ExpectEqual will test that want == got for protobufs, call t.Error if it does not,
 // and return a bool to indicate the result. This mimics the behavior of the testify `assert`
 // functions.
-func EqualProto(t *testing.T, want, got interface{}) bool {
+func ExpectEqual(t *testing.T, want, got interface{}) bool {
```

Comment:
personally I'd probably opt for `AssertEqual` for the func name to keep it consistent with the testify naming, but I'm OK with it as-is.

Reply:
That makes sense to me, I'll change.